datamol/io.py

```diff
@@ -97,24 +98,28 @@ def read_sdf(
             `as_df` is True.
         include_computed: Include computed properties in the columns.  Only relevant if
             `as_df` is True.
-        sanitize: Whether to sanitize the molecules
+        sanitize_rdkit: Whether to read the molecules with the RDKit sanitize flag.
```

Comment:
Key is not `sanitize_rdkit` but `sanitize`. Also, what would be the goal of having the initial sanitize ?

It's not checking the sdf format as far as I know and is basically calling `sanitizeMol` (https://github.com/rdkit/rdkit/blob/c02e53d7aa62dff574da7e6b2dcecfd8f81df023/Code/GraphMol/FileParsers/MolFileParser.cpp#L2982) which `dm.sanitize_mol` covers

Reply:
`dm.sanitize_mol` ditches the conformers. That's how I spotted it. Honestly I am not sure what the best way addressing that. I agree `sanitize_rdkit` looks weird.

Reply:
because there is a `to_smiles  then `to_mol` in `dm.sanitize_mol` (which is more of less required in some cases).  We should copy all molecule properties in the process, including conformers.

Reply:
Here: https://github.com/datamol-org/datamol/blob/e6225d6490482db2e327972866eab343136e8743/datamol/mol.py#L189 we should copy the propdict of input molecule, as well as conformers.  Think that would improve `sanitize_mol` a lot.

Reply:
We already copy the properties not the conformers. I actually considered it for this PR but got lazy xD I guess that's the way to go.

Reply:
Can you imagine a corner case that would fail copying conformers from a not-sane mol to a sane mol? (atom orders?)

Reply:
> We already copy the properties not the conformers. I actually considered it for this PR but got lazy xD I guess that's the way to go.

Yes in the sdf reader. Should be in sanitize as an option actually for other usage. Looking at the 
https://github.com/rdkit/rdkit/blob/c02e53d7aa62dff574da7e6b2dcecfd8f81df023/Code/GraphMol/FileParsers/MolFileParser.cpp#L2977-L3005

`sanitize=True` also assign stereochemistry, while `sanitize=False` only detects it.

Reply:
> Can you imagine a corner case that would fail copying conformers from a not-sane mol to a sane mol? (atom orders?)

If nothing is reordered or new atoms (e.g hydrogen are not added), normally no.  But in those cases where charge correction (or aromatic nitrogen correction) should not be applied